functions/commonfn.php

```diff
@@ -357,6 +361,33 @@ function amuRegisterUser(){
 		//return message
 		return $newid;
 	}
+
+    /**
+     * Contacts central auth to determine if uun exists
+     *
+     * @param string $uun
+     *
+     * @return boolean
+     */
+    function is_valid_uun( $uun ) {
+        global $amu_ldap_config;
+        $ldapconn  = ldap_connect(  $amu_ldap_config['ldaphost'],  $amu_ldap_config['ldapport'] ) or wp_die( "Could not connect to {$amu_ldap_config['ldaphost']}", 200 );
```

Comment:
Can we change this to be a logical or instead?

Reply:
done

Reply:
Actually changed this and it caused errors - not too sure why. So am sticking with or.

Reply:
Could you detail what the error is? I'm guessing the change in prescedence is what is causing the problem but it would be good to understand why this is. Would you be able to find the error and share it here just to make sure we are making the right decision? Cheers Richard.

Reply:
I look into it. Interestingly they use an or in the php manual.

Reply:
Discussed with @richeduni and he is going to get the warnings that were being produced. I can't see why these would be produced at the moment so I think it is worth fully investigating before closing out this issue.